docs/getting-started/09-trigger-microservice-with-event.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Trigger the microservice with an event
+type: Getting Started
+---
+
+This tutorial shows how to trigger the deployed `orders-service` microservice with the `order.deliverysent.v1` event from Commerce mock previously connected to Kyma.
+
+## Reference
+
+This guide demonstrates how [Event Mesh](/components/event-mesh/) works in Kyma. It allows you to receive business events from external solutions and trigger business flows with Functions or microservices.
+
+## Steps
+
+### Create the event trigger
+
+Follow these steps:
+
+<div tabs name="steps" group="trigger-microservice">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Create the [Trigger CR](https://knative.dev/docs/eventing/triggers/) for the `orders-service` microservice to subscribe it to the `order.deliverysent.v1` event type from Commerce mock:
+
+```yaml
+cat <<EOF | kubectl apply -f  -
+apiVersion: eventing.knative.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: orders-service
+  namespace: orders-service
+spec:
+  broker: default
+  filter:
+    attributes:
+      eventtypeversion: v1
+      source: commerce-mock
+      type: order.deliverysent
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: orders-service
+      namespace: orders-service
+EOF
+```
+
+- **spec.filter.attributes.eventtypeversion** points to the specific event version. In this example, it is `v1`.
+- **spec.filter.attributes.source** is taken from the name of the Application CR and specifies the source of events. In this example, it is `commerce-mock`.
+- **spec.filter.attributes.type** points to the given event type to which you want to subscribe the microservice. In this example, it is `order.deliverysent`.
+
+2. Check that the Trigger CR was created and is ready. Its status should be `True`:
+
+   ```bash
+   kubectl get trigger orders-service -n orders-service -o=jsonpath="{.status.conditions[2].status}"
+   ```
+
+   </details>
+<details>
+<summary label="console-ui">
+Console UI
+</summary>
+
+1. Select the `orders-service` Namespace from the drop-down list in the top navigation panel.
+
+2. Go to **Operation** > **Services** in the left navigation panel and select `orders-service`.
+
+3. Once in the service's details view, select **Add Event Trigger** in the **Event Triggers** section.
+
+4. Find the `order.deliverysent` event type with the `v1` version from the `commerce-mock` application. Mark it on the list and select **Add**.
+
+   A message will appear on the UI confirming that the event trigger was created, and you will see it in the **Event Triggers** section of the service's details view.
```

Comment:
`in the UI` or `on the screen`. See [the Microsoft style guide](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui).

I'd also consider restructuring it to:
```suggestion
   A message confirming that the event trigger was created will appear in the **Event Triggers** section of the service's details view.
```
or
```
   A message confirming that the event trigger was created will appear in the UI, in/around the **Event Triggers** section of the service's details view.
```